src/hook.c

```diff
@@ -433,15 +448,26 @@ static VOID EnterSpinLock(VOID)
     SIZE_T spinCount = 0;
 
     // Wait until the flag is FALSE.
+#ifndef _MSC_VER
+    while (InterlockedCompareExchange(&g_isLocked, TRUE, FALSE) != FALSE)
+    {
+#else
     while (_InterlockedCompareExchange(&g_isLocked, TRUE, FALSE) != FALSE)
     {
         _ReadWriteBarrier();
-
+#endif
         // Prevent the loop from being too busy.
+#ifndef _MSC_VER
+        if (spinCount < 16)
+            usleep(250);
```

Comment:
As far as I tried on my side. I couldn't build this due to ```usleep()```.
I think that we don't need to spend time trying to use ```usleep()```. The code below will be all right.
```
// Prevent the loop from being too busy.
if (spinCount < 32)
    Sleep(0);
else
    Sleep(1);
```